drivers/ethernet/eth_xlnx_gem.c

```diff
@@ -312,6 +312,37 @@ static void eth_xlnx_gem_isr(const struct device *dev)
 		}
 	}
 
+	/* DMA Rx buffer used bit not clear error handling */
+
+	if ((reg_val & ETH_XLNX_GEM_IXR_RX_USED_BIT) != 0x00000000U) {
+		sys_write32(ETH_XLNX_GEM_IXR_RX_USED_BIT,
+			dev_conf->base_addr + ETH_XLNX_GEM_IDR_OFFSET);
+		sys_write32(ETH_XLNX_GEM_IXR_RX_USED_BIT,
+			dev_conf->base_addr + ETH_XLNX_GEM_ISR_OFFSET);
```

Comment:
unnecessary due to lines 355-357 (324-326 pre modification), that's got the clearing of the bit covered - after the handling of the condition has occurred.